src/uct/api/uct.h

```diff
@@ -145,74 +145,76 @@ typedef struct uct_tl_resource_desc {
  *
  * The enumeration list presents a full list of operations and capabilities
  * exposed by UCX API.
+ * @{
+ */
+        /* Active message capabilities */
+#define UCT_IFACE_FLAG_AM_SHORT       UCS_BIT(0)  /**< Short active message */
```

Comment:
Doxygen assumes macro definitions as a functions in topic definition. I added one more subgroup and it looks nice at the moment. Also, this subgroup allow to set group reference.